hw/ip/rv_core_ibex/data/rv_core_ibex.hjson

```diff
@@ -23,10 +23,16 @@
       desc: "Software triggered Alert for recoverable faults",
     },
     { name: "fatal_hw_err",
-      desc: "Ibex core triggered alert for fatal faults, including integrity faults",
+      desc: '''
+        Triggered when
+          - Ibex raises ``alert_major_internal_o``
+          - Ibex raises ``alert_major_bus_o``
+          - A double fault is seen (Ibex raises ``double_fault_seen_o``)
```

Comment:
There's still some discussion about the double fault part (https://github.com/lowRISC/opentitan/issues/10523) so this may change but whatever we do it needs documenting.